tachyon/zk/plonk/vanishing/BUILD.bazel

```diff
@@ -26,13 +26,14 @@ tachyon_cc_library(
         "//tachyon/base/types:always_false",
         "//tachyon/zk/base:rotation",
         "//tachyon/zk/lookup/halo2:evaluator",
-        "//tachyon/zk/lookup/halo2:prover",
```

Comment:
Is this change relevant to this commit?

Reply:
Nope should be in separate commit, I'll fix this!